website/architecture/render-pipeline.md

```diff
@@ -188,7 +188,7 @@ After React creates the new _React Element Tree_ and _React Shadow Tree_, it mus
 
 - **Tree Promotion (Next Tree → Rendered Tree)**: This step atomically promotes the “next tree” to “previously rendered tree” so that the next mount phase computes a diff against the proper tree.
 - **Tree Diffing:** This step computes the diff between the “previously rendered tree” (**T**) and the “next tree” (**T'**). The result is a list of atomic mutation operations to be performed on _host views_.
-  - In the above example, the operations consist of: `UpdateView(**Node 3'**, {backgroundColor: '“yellow“})`
+  - In the above example, the operations consist of: `UpdateView(**Node 3'**, {backgroundColor: 'yellow'})`
```

Comment:
```suggestion
  - In the above example, the operations consist of: `UpdateView(**Node 3**, {backgroundColor: 'yellow'})`
```

Nit, looks like the single quote after node name is also redundant.

Reply:
I believe the single quote represents a prim (`3'` is a new version of `3`) throughout the whole article. Let me quickly check if it's needed here or not.